### ISSUE TITLE
def/name of "ciliary or bacterial-type flagellar motility"

### ISSUE THREAD

**Issue by gocentral** [geneontology/go-ontology#10898]
Hi,

I am correcting an annotation in MGI originally  made to the term "ciliary or bacterial-type flagellar motility". What was shown in the paper was loss of ciliary movement. However, from its def, this term should be used only when ciliary or flagellar movement is involved in cell motility, but that's not obvious from the term name. Even the names of the two child terms could possibly be made clearer. So looking at this term and it's two child terms, 

**Current terms and defs:**
GO:0001539 - ciliary or bacterial-type flagellar motility
Def: Cell motility due to movement of cilia or flagella.

GO:0071973 - bacterial-type flagellar cell motility
Def: Cell motility due to the motion of one or more bacterial-type flagella. A bacterial-type flagellum is a motor complex composed of an extracellular helical protein filament coupled to a rotary motor embedded in the cell envelope.

GO:GO:0060285 - ciliary cell motility
Def: Cell motility due to the motion of one or more cilia.

**I have some suggestions:**

1. Rename "ciliary or bacterial-type flagellar motility" to 
ciliary or bacterial-type flagellar based cell motility

2. Consider marking the term "ciliary or bacterial-type flagellar motility" as a "Do not annotate" term. Since these two structures do not ever exist in the same organism, i.e. eukaryotic cilia/flagella are microtubule based structures present only in eukaryotes while bacterial-type flagella are flagellin based structures present only in bacteria, it doesn't seem like you could ever not know which type of organism you are annotating and therefore which of the two child terms is appropriate.

3. Minor name change to the two child terms to try to make it clearer that these terms are for cell motility that depends on ciliary or flagellar movement.

A. change "bacterial-type flagellar cell motility" to "bacterial-type flagellar based cell motility"

B. change "ciliary cell motility" to "ciliary based cell motility"

4. Considering that the def of the bacterial term includes a description of the structure of the bacterial flagellum, may the ciliary term could include a description of the eukarytic cilium, including a comment that eukaryotic cilia and flagella are two names for the same structure, at least motile cilia and flagella.

thanks,

-Karen

Reported by: krchristie

Original Ticket: [geneontology/ontology-requests/10712](https://sourceforge.net/p/geneontology/ontology-requests/10712)

**Comment by paolaroncaglia** [comment]
Now reviewed. Going back to the "Things left to do for this ticket":
- check/add taxon rules for bacterial and archaeal terms: only in prokaryotes/bacteria/archaea; [done as part of https://github.com/geneontology/go-ontology/issues/11313]
- add archaeal terms to prokaryotic subset? [they have taxon rules now, so let’s leave this aside, less of a priority]
- re-house annotations to 'cilium or flagellum-dependent cell motility'; [there are now no direct, manual experimental annotations to this term. However, see below]
- add logical definitions where possible [doesn't seem necessary]

So summing up 
NOTE FOR SELF
Once the annotations have been dealt with, I can close this ticket.

**Comment by paolaroncaglia** [comment]
@mcourtot please :-)

The grouping term
GO:0001539 cilium or flagellum-dependent cell motility
(“Cell motility due to movement of eukaryotic cilia or bacterial-type flagella or archaeal-type flagella.”)
is marked not for direct manual annotation. Indeed, it doesn’t have any direct experimental annotations. However, it has 9 old NAS annotations to human, and 1 to Trypanosome. These are all UniProt. Could you please move them to ‘cilium-dependent cell motility’ (a taxonomically appropriate choice)?
I suspect the relevant proteins may have been annotated to more granular terms in the meantime, so the old NAS annotations may be superfluous, but I didn’t have time to check, and the fix I’m proposing is the quickest one.
Thanks!

**Comment by paolaroncaglia** [comment]
Actually, looking into this more closely. 

@cmungall @dosumis 

(Feel free to ignore the rest of this ticket)
Would adding the following links be appropriate:
cilium-dependent cell motility has_participant ciliated cell
and
cilium movement has_participant cilium
?
Or would occurs_in or mediated_by be more correct for the latter?

Thanks!

**Comment by paolaroncaglia** [comment]
@mcourtot please :-)

The grouping term
GO:0001539 cilium or flagellum-dependent cell motility
(“Cell motility due to movement of eukaryotic cilia or bacterial-type flagella or archaeal-type flagella.”)
is marked not for direct manual annotation. Indeed, it doesn’t have any direct experimental annotations. However, it has 9 old NAS annotations to human, and 1 to Trypanosome. These are all UniProt. Could you please move them to ‘cilium-dependent cell motility’ (a taxonomically appropriate choice)?
I suspect the relevant proteins may have been annotated to more granular terms in the meantime, so the old NAS annotations may be superfluous, but I didn’t have time to check, and the fix I’m proposing is the quickest one.

Thanks!

**Comment by mcourtot** [comment]
Annotations changes made.

**Comment by paolaroncaglia** [comment]
Thanks @mcourtot !!

**Comment by paolaroncaglia** [comment]
@cmungall and/or @dosumis please

(You may ignore the rest of this ticket)

Would adding the following links be appropriate:

cilium-dependent cell motility has_participant ciliated cell
and
cilium movement has_participant cilium
?

Or would occurs_in or mediated_by be more correct for the latter?

Thanks!

Note for self: once the above is addressed, I can close this ticket.

**Comment by paolaroncaglia** [comment]
I'll close this cilia-related ticket for now. We may come back to details later if we have a chance.
